code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -3,14 +3,15 @@ GLOBAL_LIST_INIT(hardcoded_gases, list(/datum/gas/oxygen, /datum/gas/nitrogen, /
 /proc/meta_gas_list()
 	. = subtypesof(/datum/gas)
 	for(var/gas_path in .)
-		var/list/gas_info = new(6)
+		var/list/gas_info = new(7)
```

Comment:
why did this change?

Reply:
Because I had to lengthen the list by 1 to accommodate the fusion power gas data.